client/components/oneCupcake.js

```diff
@@ -35,36 +33,18 @@ class SingleCupcake extends React.Component {
       quantity: 0,
       id: cupcake.id
```

Comment:
this looks different to me now - i remember it as "this.props.match.params.id " any spcecific reason for that change?

Reply:
on my branch it says:
  this.state = {
      quantity: 0,
      cupcakeId: this.props.match.params.id
    }

Reply:
Maybe this was just showing something deleted?

Reply:
https://github.com/fantastic-four-grace-shopper/grace-shopper/blob/Nat-Jules-Saturday/client/components/oneCupcake.js

Reply:
This is master and the previous was listed above  https://github.com/fantastic-four-grace-shopper/grace-shopper/blob/master/client/components/oneCupcake.js